src/main/java/eu/europeana/metis/sandbox/service/workflow/TransformationServiceImpl.java

```diff
@@ -41,12 +45,33 @@ public RecordInfo transform(Record record) {
     return new RecordInfo(Record.from(record, recordTransformed));
   }
 
+  @Override
+  public byte[] transform(String datasetId, String datasetName, InputStream xsltToEdmExternal,
```

Comment:
Didn't we say that the datasetName, country and language are not needed here?

Reply:
Oops :) Just removed them